Harvest4usMOBILE/app/src/main/java/com/amm/harvest4us/backend/FlaskBackendConnect.kt

```diff
@@ -20,7 +21,7 @@ open class FlaskBackendConnect : BackendConnect {
     // "adb reverse tcp:5000 tcp:5000" with the device connected
     private val adbUrl = "http://localhost:5000"
 
-    private val urlPrefix = emulatorUrl
```

Comment:
I seem to be getting 'Failed to connect to server' error when trying to log in, not sure if it's related to this change/the way you set up running the app on your phone

Reply:
Yeah it's entirely because of this change - forgot to switch it back to emulatorUrl before pushing. I'll make that fix now.